lightly_studio/src/lightly_studio/core/video/video_sample.py

```
@@ -35,6 +35,13 @@ class VideoSample(Sample):
     fps = DBField(col(VideoTable.fps))
     """Video FPS"""
 
+    def __str__(self) -> str:
```

Ruff (0.15.12)

[warning] Missing docstring in magic method

(D105)